.gitmodules

```diff
@@ -1,5 +1,5 @@
 [submodule "dotCMS/src/main/enterprise"]
 	path = dotCMS/src/main/enterprise
 	url = git@github.com:dotCMS/enterprise-2.x.git
-        branch = master
+	branch = 18860-2-nodes-same-license
```

Comment:
is this change necessary?

Reply:
No it's just to make it work locally. Going to remove this change from PR.